client/src/App.tsx

```diff
@@ -1,313 +1,43 @@
 import "./App.scss";
-import { useState, useRef, useMemo, useEffect } from "react";
+import { useState, useEffect } from "react";
 import { Header } from "./components/Header/Header";
-import PlayerScore from "./components/PlayerScore/PlayerScore";
-import { Game, GameStatusKind } from "./types/game/game.types";
-import { Player } from "./types/player/player.types";
+import { Game } from "./types/game/game.types";
 import {
   DEFAULT_ROUNDS,
   DEFAULT_PLAYERS,
-  DEFAULT_TIMEOUT,
+  PLACEHOLDER_GAME,
 } from "./constants/constants";
 import { initialiseGame } from "./service/game/game";
-import {
-  calculateRoundWinner,
-  moveTopCardToBottom,
-  chooseRandomStat,
-} from "./service/round/round";
-import GameWinner from "./components/Winner/GameWinner";
 
 import { fetchPokemonPack } from "./components/hooks/use_pack";
-
-const EMPTY_GAME = {
-  players: [],
-  totalRounds: 0,
-  currentRound: 0,
-  roundWinners: [],
-  gameStatus: "READY" as GameStatusKind,
-};
-
-const EMPTY_PLAYER = {
-  id: "",
-  name: "",
-  score: 0,
-  cards: [],
-  isCardShown: false,
-  isCardEnabled: false,
-  isHuman: false,
-};
+import GameBoard from "./components/GameBoard/GameBoard";
 
 function App() {
-  const [game, setGame] = useState<Game>(EMPTY_GAME);
-  const currentRoundRef = useRef<number>(0);
-  const currentPlayerRef = useRef<Player>(EMPTY_PLAYER);
+  const [game, setGame] = useState<Game>(PLACEHOLDER_GAME);
   const [isLoading, setIsLoading] = useState<boolean>(true);
 
   useEffect(() => {
     async function fetchPack() {
       const pack = await fetchPokemonPack();
       setIsLoading(false);
+      setIsLoading(false);
```

Comment:
does this need to be in twice?

Reply:
It does not, that's from a merge, will remove